src/v3/src/components/Widget/index.tsx

```diff
@@ -78,6 +77,8 @@ import Form from '../Form';
 import IdentifierContainer from '../IdentifierContainer';
 import Spinner from '../Spinner';
 
+import './style.css';
```

Comment:
We actually no longer generate _any_ extracted styles. However we face an issue where the loginpage always expects a CSS file and we won't want to generate `404`s on loading the login page. This import of an empty file causes webpack to still emit an empty CSS file to avoid this issue for now. Would like to explore the best way to handle this long term.